lib/logstash-input-http_jars.rb

```diff
@@ -0,0 +1,7 @@
+# AUTOGENERATED BY THE GRADLE SCRIPT. DO NOT EDIT.
+
+require 'jar_dependencies'
+require_jar('io.netty', 'netty-all', '4.1.19.Final')
```

Comment:
Can you roll this back to 4.1.18.Final. 

Beats, TCP and now this will all need to agree upon the version in use. 

https://github.com/logstash-plugins/logstash-input-beats/blob/master/build.gradle#L25

IIRC the mismatch caused an issue with TLS: https://github.com/logstash-plugins/logstash-input-tcp/pull/113 (it may not present here, but best to be safe)